apollo-core/README.md

```diff
@@ -79,10 +79,10 @@ using `-D` command-line options.  For example, `-Ddomain=example.org`
 sets the `domain` config key to `example.org`.
 
 Apollo Core also considers environment variables of the form
-`SPOTIFY_X_Y=ab`.  These are translated into configuration keys
-of the form `x.y=ab`.  The values of the environment variables are
-treated as strings and copied verbatim; no special syntax is
-supported.
+`APOLLO_X_Y=ab`, where the `APOLLO` prefix is configurable.
+These are translated into configuration keys of the form `x.y=ab`.
+The values of the environment variables are treated as strings and
+copied verbatim; no special syntax is supported.
```

Comment:
I noticed in a test that there seems to be special magic applied to underscores. Could be a good idea to explain how that works?